rviz_rendering/src/rviz_rendering/ogre_logging.cpp

```diff
@@ -124,6 +127,7 @@ void OgreLogging::configureLogging()
   if (preference_ == StandardOut) {
     ll.min_lml = Ogre::LML_NORMAL;
   }
+  delete log_manager;
```

Comment:
I don't believe this to be correct either. I think the logger will be held by `Ogre::LogManager::getSingletonPtr()`afterwards.

Reply:
Looking into it, you're absolutely right. I find this quite weird style, thouh: I would've expected there to be some kind of static factory method - if I `new` an object, this should mean I'm also responsible to `delete` it. I guess we have to disable cppcheck for this `new`call then. I'll look into it.